student.library/pom.xml

```diff
@@ -45,6 +53,23 @@
 			<version>${item-scoring.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.opentestsystem.common</groupId>
+			<artifactId>tds-common-legacy</artifactId>
+			<version>${tds-common-legacy.version}</version>
+		</dependency>
+
+		<dependency>
```

Comment:
I will remove this dependency.  I was using it for message translation but am no longer doing that here.